### DISCUSSION TITLE
Using an external GPS with Qfield

### DISCUSSION THREAD

**Discussion by careyshan** [opengisch/QField#2676] [Q&A]
Hi All,
I was wondering what is the best external GPS unit to use with Qfield?
Thanks

**Comment by lindacamathias** [comment]
Hi @careyshan 
i tested reach rs2 from emlid successfully and i know that the rs2 is used from different people in combination with QField too.  Tests with other devices are next on my agenda. i'll let you know about my results. Potentially all devices which can send a NMEA protocoll through bluetooth should be able to connect to QField directly.

**Reply by lindacamathias** [reply]
Leica Zeno FLX100 ZM does also work with QField. Further information and tests to come ..

**Reply by Marwe** [reply]
We have REACH RS2 and M2 here, both work fine, though we found on our devices (tablet samsung active tab 3), that TCP connections might be more stable at least with RS2 (bluetooth issues with android and firmware, recent updates may have fixed some issues), in that case we use the Lefebure NTRIP app additionally and provide mock locations to the system (enable android developer settings, then set mock location app in developer options). The M2 currently has a few issues with getting NTRIP and sending TCP via WLAN Hotspot on tablet.
QField is recommended to connect via bluetooth directly (see comment of @lindacamathias), so if your bluetooth connection is stable, go for that (more metadata from NMEA than system).

**Comment by careyshan** [comment]
Brilliant - that is great info to know. Thanks @lindacamathias

**Comment by geomondego** [comment]
Hello @careyshan,
In the past i used a Trimble Catalyst with mock location via Trimble Manager app and it worked great in QField. I know that this antenna now also has a bluetooth option, but still haven't got one for testing

**Comment by MH-Mueller** [comment]
Hello,

we are using a "Leica GS18 T GNSS RTK Rover" with a "Leica CS35 Captivate tablet" (windows). We installed the "windows-version" of QField. I'm trying to connect the antenna to QField via Bluetooth, but I was not succesful yet. Has anyone experience with this equipment? Theoretically the RTK Rover can send the NMEA through bluetooth and the NMEA sentences correspond to those which can be read by QField. However, in the positioning settings of QField I cannot find the device.

Any help is appreciated.

Thanks

Matthias

**Reply by nirvn** [reply]
@MH-Mueller , I do not have this device with me, however as a generic statement, Bluetooth connection to external antennas on windows go through a virtual serial port created by the OS. You do that by connecting to your antenna via Bluetooth using Windows itself, which will create a serial port QField can sue (Windows instructions here: https://junipersys.com/support/article/14876). Once connected, you can open QField, go to the settings' positioning tab and click on the ( add ) button. Then, pick up the serial port receiver type, and choose the right COM (usually COM3 or COM4).

**Reply by MH-Mueller** [reply]
@nirvn, thanks for your advices! Now it's working.
